### ISSUE TITLE
bug: Symfony.ControlStructure.UnaryOperators detect non adjacent unary increment / decrement  operatos 

### ISSUE THREAD

**Issue by xalopp** [djoos/Symfony-coding-standard#116]
Have a look at:

```php
<?php
// currently fails, but 
// https://github.com/djoos/Symfony-coding-standard/issues/94#issuecomment-324047371
// says it should pass
$x++;

// passes, but should fail
--     $x;
--
    $x;
--/* not adjacent */
    $x;
--// not adjacent
    $x;
```

Scanning this file with Symfony Conding Standard, only points out that  `$x++` is not adjacent to it's variable. This should be in my opinion adjacent, as pointed out by @razbakov  in #94.

All not adjacent occurrences, are unfortunately not properly detected at all. This makes this Sniff actually quite useless. 😞  

I'm already working on this issue, so I hope to be able to contribute a fix soon 😄

**Comment by istrof** [comment]
Please review this PR, it should fix this problem:

https://github.com/djoos/Symfony-coding-standard/pull/125

**Comment by djoos** [comment]
#125 merged!

**Comment by xalopp** [comment]
thx @istrof & @djoos
